README.md

```diff
@@ -1,83 +1,80 @@
-Scel
-====
+# scel - sclang-mode for emacs
 
 SuperCollider/Emacs interface
 
+## Installation
 
-Installation requirements
--------------------------
+The repository contains two subprojects. `/sc` contains the SuperCollider code
+required to implement the emacs interface. `/el` contains the emacs-lisp
+implementation of the mode. Emacs and SuperCollider each have their own package
+managers, so it is required to install each half separately.
 
-For the HTML help system, you will need emacs-w3m support.
+### Installing scel quark
 
+The `scel` Quark is required for emacs to communicate with sclang. Evaluate this in the SuperCollider GUI:
 
-Installation (default)
-----------------------
-
-By default emacs-lisp files are installed in
-
-`$prefix/share/emacs/site-lisp`
-
-SuperCollider files are put in
+``` supercollider
+Quarks.install("https://github.com/supercollider/scel");
+```
 
-`$prefix/share/SuperCollider/Extensions/scide_scel`
+### Installing the emacs mode
 
+Using straight.el
+``` emacs-lisp
+(straight-use-package
+ '(sclang :type git 
+          :host github 
+          :repo "supercollider/scel" 
+          :files ("el/*.el")))
+```
 
-The only thing you need to do is loading the sclang interface in your `~/.emacs`:
+Or download the repo directly to your user config directory
 
+``` shell
+git clone https://github.com/supercollider/scel.git ~/.emacs.d/scel
 ```
+``` emacs-lisp
+(add-to-list 'load-path "~/.emacs.d/scel/el/")
 (require 'sclang)
 ```
 
-For the HTML help system to fully function also add
-```
-(require 'w3m)
+### On MacOS
+
+If `sclang` executable is not on your path, you may need to add it to your exec-path.
+
+``` emacs-lisp
+(setq exec-path (append exec-path '("/Applications/SuperCollider.app/Contents/MacOS/")))
 ```
+### On Linux
 
+If you are building SuperCollider from source on Linux, this library (both .el
+and .sc files) will be installed by default. To disable it pass the flag
+`-DSC_EL=OFF` as a `cmake` option. See the supercollider readme for more info.
```

Comment:
Maybe the fact that we have two ways of installing scel should be elaborated a little more - possibly not in this section though, but in the Introduction. Also I don't think that CMake install technically Linux-only (thought I might be wrong). But yes, it is the current way to get it on Linux, so I think we could say that it could be installed using either method - when compiling, OR using the new method (but not both! - important to spell that out)

Reply:
Also, debian/ubuntu (maybe other distros too?) provide supercollider-emacs package. It might be worth noting that CMake install is equivalent to the system package install and should not be performed together with the "new" install method.